test/Auctioneer.test.js

```diff
@@ -51,7 +51,10 @@ describe("Auctioneer", () => {
   })
 
   describe("createAuction", () => {
+    let previousAuctionsCount
```

Comment:
Instead of noting it, can't we change the main `before` into `beforeEach` so that each test is independent? I belive we use this pattern in every other test file.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/571dd99fb00e43b4e6bfb1b08beb50ecd05f7a23